web/index.go

```diff
@@ -504,8 +516,16 @@ var indexHTML = []byte(`
         data,
         {
           labels: labels,
+          labelsSeparateLines: true,
+          labelsKMG2: true,
           includeZero: true,
-          legend: 'always',
+          legend: 'follow',
```

Comment:
One downside of this change is that the legend won't show by default on ops panel. It causes no trouble to me as the first reaction to unusual graphs is hovering to see the actual data. A custom `legendFormatter` can be used if always showing legend is a requirement.